var/spack/repos/builtin/packages/pixman/package.py

```diff
@@ -62,6 +62,8 @@ def configure_args(self):
         args = ["--enable-libpng", "--disable-gtk"]
 
         if sys.platform == "darwin":
+            # neither style of acceleration work on macOS
```

Comment:
The change LGTM, but can you put the links in the PR description in the comment above?

Reply:
done!